doc/docs/learn/create-an-app-with-monomer.md

```diff
@@ -0,0 +1,568 @@
+---
+sidebar_position: 2
+---
+
+# Create an Application With Monomer
+
+In this tutorial, you will learn how to create a new Monomer application with
+a Cosmos SDK [simapp](https://docs.cosmos.network/main/learn/intro/sdk-design#baseapp) (or, `baseapp`).
+
+## Setup
+
+Monomer is a framework for deploying Cosmos SDK applications as rollups on
+Ethereum. As such, the development environment for Monomer is similar to that of
+a Cosmos SDK application. To begin building an application with Monomer, we can
+start with a basic Cosmos SDK app. 
+
+We'll use the handy tool [spawn](https://github.com/rollchains/spawn) to bootstrap a new Cosmos SDK application. If you do not already have it, follow the instructions below to install it.
+
+<details>
+  <summary>Install `spawn`</summary>
+
+```bash
+git clone https://github.com/rollchains/spawn
+cd spawn
+git checkout v0.50.4
+make install
+```
+</details>
+
+
+### Bootstrapping a Cosmos SDK Application
+We can spawn a new Cosmos SDK application with `spawn new`:
+```bash
+spawn new rollchain --consensus=proof-of-authority \
+    --bech32=roll \
+    --denom=uroll \
+    --bin=rolld \
+    --disabled=cosmwasm,globalfee \
+    --org=monomer
+```
+
+This gives us a boilerplate Cosmos SDK application in the `rollchain`
+directory.
+
+## Integrating Monomer
+Let's move into the `rollchain` directory and add Monomer to our application.
+
+```bash
+cd rollchain
+go get github.com/polymerdao/monomer
+```
+
+### Import `x/rollup` and `x/testmodule`
+
+Now we can import [`x/rollup`](/docs/learn/the-rollup-module) in `app/app.go`. While we're at it, let's also import `x/testmodule` (this
+will [initialize a non-empty validator set](https://github.com/polymerdao/monomer/blob/c98eccb49bf857829cadee899359e60fc36e6745/testapp/x/testmodule/module.go#L82) for us so we don't have to do it manually). When you're ready to deploy your application, you can remove `x/testmodule` and configure your own validator set.
+```go
+import (
+    // ...
+    "sync"
+
+    "github.com/polymerdao/monomer/x/rollup"
+    rolluptypes "github.com/polymerdao/monomer/x/rollup/types"
+    rollupkeeper "github.com/polymerdao/monomer/x/rollup/keeper"
+    "github.com/polymerdao/monomer/testapp/x/testmodule"
+    testmodulekeeper "github.com/polymerdao/monomer/testapp/x/testmodule"
+    // ...
+)
+```
+
+There are a few modifications we need to make to the boilerplate Cosmos SDK app. Namely, we need to initialize the module's store keys, add the module to the genesis module order, and initialize the module in `app/app.go`. Let's walk through these changes.
```

Comment:
Better to avoid long paragraphs. 
```suggestion
There are a few modifications we need to make to the boilerplate Cosmos SDK app.

1. Namely, we need to initialize the module's store keys
2. Add the module to the genesis module order
3. Initialize the module in `app/app.go`.

Let's walk through these changes.
```